src/themes/Treedip/DonationForm/SubComponents/SelectGroupRadioBubbles.scss

```diff
@@ -0,0 +1,67 @@
+@use '../../variables/colors';
+@use '../../variables/breakpoints';
+
+$height: 15px !default;
+$width: 15px !default;
+$margin: -6px 0 0 0 !default;
+$border: 1px solid #bbbbbb !default;
+$margin-left: none !default;
+$margin-right: 0 !default;
+$border-radius: 4px !default;
+$border-radio: 6px solid colors.$primary !default;
+$box-shadow: none !default;
+$background-color: transparent !default;
+$height-label: 21px !default;
+$font-size: 0.095em !default;
+$padding: 0 0 !default;
+$border-label: 2px solid #58575717 !default;
+
+.wmde-banner {
+	&-select-group {
+		&-option label {
```

Comment:
To put a border around the label when its checkbox is selected, you'll need to add a class in the Vue file. For non-bubble components that class won't do anything, so it'll be backwards compatible. Preferrably add the `active` class to the `wmde-banner-select-group-option` (the element that contains the `v-for`).
The selector for that will then be underneath these definitions, something like `&-option.active label` (mind the missing dot before active, it means that the element must have *both* classes)

The reason why we have to do this in the Vue file and not in CSS is the logic of hierarchy in CSS selectors. We always say "I want to style an element that has the following parents", e.g. styling a `label` that's a child of a parent that has the `active` class: `.active label` 
Up until recently, it has been impossible to express a rule for a parent element that depends on some class or pseudoclass of a child. If we had something like a "parent" selector, let's pretend it would be "<", the you could express something like "styling a label element that has a checked checkbox" with a selector like `label < input:checked`. See this StackOverflow question form 2009: https://stackoverflow.com/questions/1014861/is-there-a-css-parent-selector
With the upcoming [:has selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:has) styling parents based on children will be possible, but that selector is quite new and not supported in Firefox. The selector would then look like `label:has(input:checked)`